core/settings/qPrefAnimations.h

```diff
@@ -18,7 +19,7 @@ class qPrefAnimations : public QObject {
 	void sync() { loadSync(true); }
 
 public:
-	int animation_speed() const;
+	static inline int animation_speed() { return prefs.animation_speed; };
```

Comment:
same here as the last comment.
And, BTW, from a white space perspective, I like this version with a space inside the `{ ` and ` }` better than the `{return...` white space in the earlier commits.
I guess this is something I should add to coding style now that we have more instances of `{` not being followed by a newline...